libs/cardano-ledger-core/src/Cardano/Ledger/BaseTypes.hs

```diff
@@ -853,3 +855,8 @@ class Inject t s where
   inject :: t -> s
   default inject :: t ~ s => t -> s
   inject = id
+
+-- | Helper function for a common pattern of creating objects
+kindObject :: Text -> [Pair] -> Value
+kindObject name obj =
+  object $ ["kind" .= name] ++ obj
```

Comment:
Couldn't you use `("kind" .= name) : obj`  here  instead of making a list and concatenating it?

Reply:
Yes. Nice catch